tap_typeform/streams.py

```diff
@@ -271,7 +270,7 @@ def sync_forms(atx):
                 [responses, max_submitted_at] = sync_form(atx, form_id, ut_interim_next_date, ut_next_date)
 
             # if the prior sync is successful it will write the date_to_resume bookmark
-            write_forms_state(atx, form_id, next_date)
+            write_forms_state(atx, form_id, pendulum.parse(max_submitted_at) + datetime.timedelta(seconds=1))
```

Comment:
Singer has a convention to be inclusive with date windows. We run the risk of missing data otherwise.

Reply:
Thanks for the review @luandy64 , I've updated this and removed the +1 seconds part. Please let me know if this addresses your concern!